components/kyma-environment-broker/cmd/broker/deprovisioning_suite_test.go

```diff
@@ -0,0 +1,236 @@
+	avsConfig := avs.Config{
+		OauthTokenEndpoint: fmt.Sprintf("%s/oauth/token", mockServer.URL),
+		ApiEndpoint:        fmt.Sprintf("%s/api/v2/evaluationmetadata", mockServer.URL),
+	}
+	client, err := avs.NewClient(context.TODO(), avsConfig, logrus.New())
+	assert.NoError(t, err)
+	_, err = client.CreateEvaluation(&avs.BasicEvaluationCreateRequest{
+		Name: "fake-evaluation",
+	})
+	assert.NoError(t, err)
+	avsDel := avs.NewDelegator(client, avsConfig, db.Operations())
+	externalEvalAssistant := avs.NewExternalEvalAssistant(cfg.Avs)
+	internalEvalAssistant := avs.NewInternalEvalAssistant(cfg.Avs)
+
+	smcf := servicemanager.NewFakeServiceManagerClientFactory(nil, nil)
+
+	iasFakeClient := ias.NewFakeClient()
+	bundleBuilder := ias.NewBundleBuilder(iasFakeClient, cfg.IAS)
+
+	edpClient := fixEDPClient()
+
+	namespaceClientResourceGroupDoesNotExist := azuretesting.NewFakeNamespaceClientResourceGroupDoesNotExist()
+	hyperscalerProvider := azuretesting.NewFakeHyperscalerProvider(namespaceClientResourceGroupDoesNotExist)
+	accountProvider := fixAccountProvider()
+
+	deprovisionManager := deprovisioning.NewManager(db.Operations(), eventBroker, logs.WithField("deprovisioning", "manager"))
+
+	deprovisioningQueue := NewDeprovisioningProcessingQueue(ctx, workersAmount, deprovisionManager, cfg, db, eventBroker,
+		provisionerClient, avsDel, internalEvalAssistant, externalEvalAssistant, smcf,
+		bundleBuilder, edpClient, hyperscalerProvider, accountProvider, logs,
+	)
+
+	return &DeprovisioningSuite{
+		provisionerClient:   provisionerClient,
+		deprovisioningQueue: deprovisioningQueue,
+		storage:             db,
+
+		t: t,
+	}
+}
+
+func (s *DeprovisioningSuite) CreateProvisionedRuntime(options RuntimeOptions) string {
+	randomInstanceId := uuid.New().String()
+
+	instance := fixture.FixInstance(randomInstanceId)
+	instance.GlobalAccountID = options.ProvideGlobalAccountID()
+	instance.SubAccountID = options.ProvideSubAccountID()
+	instance.InstanceDetails.SubAccountID = options.ProvideSubAccountID()
+	instance.Parameters.PlatformRegion = options.ProvidePlatformRegion()
+	instance.Parameters.Parameters.Region = options.ProvideRegion()
+	instance.ProviderRegion = *options.ProvideRegion()
+
+	provisioningOperation := fixture.FixProvisioningOperation(operationID, randomInstanceId)
+
+	require.NoError(s.t, s.storage.Instances().Insert(instance))
+	require.NoError(s.t, s.storage.Operations().InsertProvisioningOperation(provisioningOperation))
+
+	state, err := s.provisionerClient.ProvisionRuntime(options.ProvideGlobalAccountID(), options.ProvideSubAccountID(), gqlschema.ProvisionRuntimeInput{})
+	require.NoError(s.t, err)
+
+	s.finishProvisioningOperationByProvisioner(gqlschema.OperationTypeProvision, *state.RuntimeID)
+
+	return instance.InstanceID
+}
+
+func (s *DeprovisioningSuite) finishProvisioningOperationByProvisioner(operationType gqlschema.OperationType, runtimeID string) {
+	err := wait.Poll(time.Millisecond*100, 15*time.Second, func() (bool, error) {
```

Comment:
reduce all duration to have a possibility to finish the test about 10ms, see current provisioning values

Reply:
add also the "Speedup" factor

Reply:
done